packages/report-server/src/reportBuilder/reportBuilder.ts

```diff
@@ -4,31 +4,33 @@
  */
 
 import { Aggregator } from '../aggregator';
-import { FetchReportQuery, ReportConfig } from '../types';
+import { FetchReportQuery } from '../types';
 import { buildFetch } from './fetch';
 import { buildTransform } from './transform';
 import { buildOutput } from './output';
 import { Row } from './types';
 import { OutputType } from './output/functions/outputBuilders';
 
-interface BuildReport {
+interface BuiltReport {
   results: OutputType;
 }
 
 export class ReportBuilder {
-  config?: ReportConfig;
+  config?: Record<string, unknown>;
 
   testData?: Row[];
 
-  setConfig = (config: ReportConfig) => {
+  setConfig = (config: Record<string, unknown>) => {
```

Comment:
Changing this to `Record<string, unknown>` is valid as each builder runs validation on the config anyway, and the config at this point may truly be unknown